### DISCUSSION TITLE
is there any way to capture the Dataview plugin's output and statisize it during export?

### DISCUSSION THREAD

**Discussion by jcolson** [zoni/obsidian-export#75] [Support and Q&A] [answered]
i have quite a few pages that use the dataview plugin to put summary tables together.  it would be great if there was someway to make those pages static so that it is output during the export.

**Comment by zoni** [comment, marked as answer]
_Theoretically_, it would be possible to write a [postprocessor](https://docs.rs/obsidian-export/latest/obsidian_export/type.Postprocessor.html) which reimplements dataview functionality in Rust. In practice, I fear dataview has such a large API that this would be a non-starter.

So the realistic answer is, no, this is not possible unfortunately. Exporting notes with all the result/output of plugins intact would really have to be something implemented natively in Obsidian itself.

**Comment by jcolson** [comment]
understood ... next stop -> i'll try to see if there is a way for dataview to statically render pages somewhere, and THEN use obsidian-export ....

**Reply by aadimator** [reply]
Let me know if you find anything? I've been looking for this as well.

**Reply by filmgal** [reply]
@jcolson There is a way to create a 'static' table from your dataview table using a templater template.  See https://forum.obsidian.md/t/dataviewjs-snippet-showcase/17847/225?u=lise
